implementation/irq-closures.tex

```diff
@@ -86,7 +86,10 @@ \subsection{Motivation}
 
 \subsection{Implementation}
 
-This section looks at how we can achieve the same behavior as was presented in the previous section.
+This section looks at how we built an abstraction exposing the behavior that was presented in the previous section.
+The implementation shows how we can the {\unsafe} keyword to build a this safe abstraction.
```

Comment:
"The implementation shows how we can `use` the unsafe keyword to build ~~a~~ this safe abstraction"?

Legg til `use` og fjern overflødig `a`